### DISCUSSION TITLE
Upgrading from 2.4

### DISCUSSION THREAD

**Discussion by egoitzau** [Z-Hub/Z-Push#46] [General]
Hi!

I have a Z-push env in 2.4 which I would like to upgrade to lastest stable version (2.7).

I use memcached for ipc and mysql for states. 

I have been trying to find the appropiate way of performing it but I guess I should just extract the tar with the new file¿?. I have not seen nothing explicitily explained for upgrading. Should I do it that way?.

Cheers!

**Comment by matidau** [comment]
Hi @egoitzau

That is how I've been testing upgrades from 2.6.4. Make sure you make a copy of the Z-Push, Autodiscover and Backend configuration files before extracting.

Cheers,
Mat

**Comment by egoitzau** [comment]
Thank you so much!!
